src/Analyzers/CSharp/CodeFixes/CSharpCodeFixesResources.resx

```diff
@@ -187,4 +187,7 @@
   <data name="Place_token_on_following_line" xml:space="preserve">
     <value>Place token on following line</value>
   </data>
+  <data name="Make_property_required" xml:space="preserve">
+    <value>Make property required</value>
```

Comment:
```suggestion
    <value>Make property 'required'</value>
```

add a "lock" comment as well asking the localizers to not localize "required".  thanks!

Reply:
Sorry, I'm not understanding the reasoning here. 'required' should be locked since we want that keyword kept in that state in all localizations.   

We've def missed some other cases where we should be locking as well. 

I also don't see an issue with the text becoming: Add 'required' modifier. 

It's simple and clear.

Reply:
Alternatively. We can just have "Add '{0}' modifier". And then use that single loc string for all these features. Then no need for locking.